.tool-versions

```diff
@@ -0,0 +1,2 @@
+erlang 23.2.7.5
```

Comment:
I don't think we want this file. Let each dev work on their own Elixir version.

Reply:
Done